ops/services/alerts/app_service_metrics/uptime.tf

```diff
@@ -81,3 +83,35 @@ resource "azurerm_monitor_metric_alert" "uptime" {
     ]
   }
 }
+
+resource "azurerm_monitor_metric_alert" "uptime_redirects" {
+  for_each = var.additional_uptime_test_urls
+
+  name                = "uptime_redirects-${each.key}"
+  description         = "${each.key} is not responding"
+  resource_group_name = var.rg_name
+  scopes              = [azurerm_application_insights_web_test.uptime[each.key].id, var.app_insights_id]
+  frequency           = "PT1M"
```

Comment:
not sure if I am reading this correctly but currently looks like [prod-simplereport-gov-www-redirect](https://portal.azure.com/#@cdc.onmicrosoft.com/resource/subscriptions/7d1e3999-6577-4cd5-b296-f518e5c8e677/resourceGroups/prime-simple-report-prod/providers/Microsoft.Insights/webtests/prod-simplereport-gov-www-redirect/availability) and [prod-simplereport-gov-cdc-redirect](https://portal.azure.com/#@cdc.onmicrosoft.com/resource/subscriptions/7d1e3999-6577-4cd5-b296-f518e5c8e677/resourceGroups/prime-simple-report-prod/providers/Microsoft.Insights/webtests/prod-simplereport-gov-cdc-redirect/availability) availability tests the frequency is set to 5 minutes... Do we want to make this change to be every minute? 🤔

Reply:
If I'm understanding it correctly, the existing metric alerts all have a `frequency` of 1 minute but a `window size` of 5 minutes. So for these additional redirect urls the alerts will still test the uptime every minute but each of those attempts has up to 15 minutes to succeed before the alert officially fails

Reply:
OO ok got it! Thank you!